docs/src/model_setup/output_writers.md

```diff
@@ -170,7 +170,7 @@ Write out 3D fields for w and T and a horizontal average:
 using Oceananigans, Oceananigans.OutputWriters, Oceananigans.Fields
 using Oceananigans.Utils: hour, minute
 
-model = NonhydrostaticModel(grid=RectilinearGrid(size=(1, 1, 1), extent=(1, 1, 1)))
+model = NonhydrostaticModel(grid=RectilinearGrid(size=(1, 1, 1), extent=(1, 1, 1)), buoyancy=SeawaterBuoyancy(), tracers=(:T, :S))
```

Comment:
Same here. Why not just create the default NonhydrostaticModel? Do we really need T & S?